CaloReco/src/CaloRawWFProcessor.cc

```diff
@@ -63,29 +63,26 @@ namespace mu2e {
    void CaloRawWFProcessor::extract(const std::vector<double> &xInput, const std::vector<double> &yInput)
    {
 
-      reset();
+       reset();
 
-      xvec_ = xInput;
-      yvec_ = yInput;
+       xvec_ = xInput;
+       yvec_ = yInput;
 
 
-      //find location of potential peaks
-      std::vector<int> peakLocation;
-
-      for (unsigned int iu=windowPeak_;iu<xvec_.size()-windowPeak_;++iu)
-      {
-	 auto maxp = std::max_element(&yvec_[iu-windowPeak_],&yvec_[iu+windowPeak_+1]);
-	 if (maxp == &yvec_[iu] && yvec_[iu] > minPeakAmplitude_) peakLocation.push_back(iu);
-      }
-      int nPeak = peakLocation.size();
-
-      if (diagLevel_ > 1) std::cout<<"[CaloRawWFProcessor] Peaks found : "<<nPeak<<std::endl;
+       //find location of potential peaks
```

Comment:
The Raw processor can be used in the CaloRecoDigi module. The digi has already in it the position of the peak, so I think that we should not do any other loop to re-search for the peak of the waveform. Am I missing anything?